app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java

```diff
@@ -49,18 +49,15 @@
 public class SpeechFragment extends Fragment implements SpeechCallback,
         RedditSearcher.CommentCallback {
     private OnFragmentInteractionListener mListener;
+    private static final String ERROR_TAG = "SpeechFragment Error";
     private static final String DEBUG_TAG = "SpeechFragment Debug";
     private boolean isListening;
     private boolean shakeOn = true;
-    private SpeechListener listener;
-    private ArrayList voiceInput;
```

Comment:
If you can store something as a local variable you should. Otherwise it clutters fields.